cypress/integration/DropdownMenu.spec.ts

```diff
@@ -0,0 +1,280 @@
+describe('DropdownMenu', () => {
+  describe('given submenu user', () => {
+    beforeEach(() => {
+      cy.visitStory('dropdownmenu--submenus');
+    });
+
+    describe('when using pointer', () => {
+      beforeEach(() => {
+        cy.findByText('Open').click();
+      });
+
+      it('should open submenu and not focus first item when moving pointer over trigger', () => {
+        pointerOver('Bookmarks →')
+          .then(getAssociatedSubmenu)
+          .children()
+          .first()
+          .should('not.be.focused');
+      });
+
+      it('should not close when moving pointer to submenu and back to parent trigger', () => {
+        pointerOver('Bookmarks →');
+        pointerOver('Inbox');
+        pointerOver('Bookmarks →');
+        cy.findByText('Inbox').should('be.visible');
+      });
+
+      it(
+        'should close submenu when moving pointer away but remain open when moving towards',
+        {
+          viewportWidth: 550,
+        },
```

Comment:
Passing in a slimmer width to trigger the collision detection, I found this to be more robust than calling [cy.viewport](https://docs.cypress.io/api/commands/viewport) from within the body